plc4j/api/src/main/java/org/apache/plc4x/java/api/messages/PlcWriteResponse.java

```diff
@@ -37,6 +37,7 @@ public PlcWriteResponse(PlcWriteRequest request, List<? extends WriteResponseIte
 
     @SuppressWarnings("unchecked")
     public <T> Optional<WriteResponseItem<T>> getValue(WriteRequestItem<T> item) {
-        return (Optional) super.getValue(item);
+        Optional<WriteResponseItem<?>> value = super.getValue(item); 
+        return (Optional<WriteResponseItem<T>>) ((Object) value); 
```

Comment:
This cast to Object and then a type again is a bit confusing. Is this just to silence a warning?

Reply:
Correct.

Reply:
Fwiw, actually, directly casting to Optional<WriteResponseItem<T>> is an error.  First casting to Object eliminates it. Will add comment.